"build" feature fails to link avresample

I'm building on macOS 10.13 using ffmpeg-sys 8a6d9c6eecddc4c95c63977378c490196c19ea45

```toml
[dependencies.ffmpeg]
git = "https://github.com/pornel/rust-ffmpeg.git" # that pulls ffmpeg-rs master with ffmpeg-sys master
features = ["build"]
```

and crate a binary like this:

```rust
extern crate ffmpeg;
fn main() {ffmpeg::init().unwrap()}
```

cargo build fails with an error:

```
 = note: Undefined symbols for architecture x86_64:
            "_avresample_get_delay", referenced from:
                _filter_frame in libffmpeg_sys-e10cd590959afb4b.rlib(af_resample.o)
            "_avresample_get_out_samples", referenced from:
                _filter_frame in libffmpeg_sys-e10cd590959afb4b.rlib(af_resample.o)
                _request_frame in libffmpeg_sys-e10cd590959afb4b.rlib(af_resample.o)
            "_avresample_close", referenced from:
                _uninit in libffmpeg_sys-e10cd590959afb4b.rlib(af_resample.o)
                _config_output in libffmpeg_sys-e10cd590959afb4b.rlib(af_resample.o)
            "_avresample_convert", referenced from:
                _filter_frame in libffmpeg_sys-e10cd590959afb4b.rlib(af_resample.o)
                _request_frame in libffmpeg_sys-e10cd590959afb4b.rlib(af_resample.o)
            "_avresample_available", referenced from:
                _filter_frame in libffmpeg_sys-e10cd590959afb4b.rlib(af_resample.o)
            "_avresample_alloc_context", referenced from:
                _config_output in libffmpeg_sys-e10cd590959afb4b.rlib(af_resample.o)
            "_avresample_free", referenced from:
                _uninit in libffmpeg_sys-e10cd590959afb4b.rlib(af_resample.o)
                _config_output in libffmpeg_sys-e10cd590959afb4b.rlib(af_resample.o)
            "_avresample_open", referenced from:
                _config_output in libffmpeg_sys-e10cd590959afb4b.rlib(af_resample.o)
            "_avresample_get_class", referenced from:
                _init in libffmpeg_sys-e10cd590959afb4b.rlib(af_resample.o)
                _resample_child_class_next in libffmpeg_sys-e10cd590959afb4b.rlib(af_resample.o)
          ld: symbol(s) not found for architecture x86_64
          clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

--- a/build.rs
+++ b/build.rs
@@ -457,6 +457,9 @@ fn main() {
         if env::var("CARGO_FEATURE_AVDEVICE").is_ok() {
             println!("cargo:rustc-link-lib={}=avdevice", ffmpeg_ty);
         }
+        if env::var("CARGO_FEATURE_AVRESAMPLE").is_ok() {
+            println!("cargo:rustc-link-lib={}=avresample", ffmpeg_ty);
+        }
         if env::var("CARGO_FEATURE_SWSCALE").is_ok() {
             println!("cargo:rustc-link-lib={}=swscale", ffmpeg_ty);
         }
@@ -525,6 +528,11 @@ fn main() {
         pkg_config::Config::new()
             .statik(statik)
             .probe("libavdevice")
+            .unwrap();
+
+        pkg_config::Config::new()
+            .statik(statik)
+            .probe("libavresample")
             .unwrap();
 
         pkg_config::Config::new()

